pcsx2/gui/GlobalCommands.cpp

```diff
@@ -184,15 +184,15 @@ namespace Implementations
 			default: break;
 		}
 
-		Console.WriteLn(L"(GSwindow) Aspect ratio: %s.", WX_STR(arts));
+		OSDlog( Color_StrongBlue, true, "(GSwindow) Aspect ratio: %s.", WX_STR(wxString(arts)));
```

Comment:
Don't use WX_STR here - change `arts` so it's a `const char*` and adjust the strings - that way you can just use `arts` directly. That should work fine on both Windows and Linux when combined with the `format.c_str()` change I mentioned in another comment.